File: Programmers/Level1/220817/javascript/Descending.test.js
import { descending, solution } from './Descending';

test('descending', () => {
  expect(descending(118372)).toStrictEqual([8, 7, 3, 2, 1, 1]);
});

test('answer', () => {
  expect(solution(118372)).toStrictEqual(873211);
});

File: Programmers/Level1/220817/java/DescendingTest.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class DescendingTest {
  @Test
  void descending() {
    Descending descending = new Descending();

    assertArrayEquals(new long[]{8, 7, 3, 2, 1, 1},
        descending.descendingArray(118372, new long[6]));
  }

  @Test
  void solution() {
    Descending descending = new Descending();

    assertEquals(873211, descending.solution(118372));
  }
}
